test/mock-utils.js

```diff
@@ -242,3 +243,33 @@ describe('buildHeadersFromArray', () => {
     t.strictEqual(headers.key, 'value')
   })
 })
+
+describe.only('normalizeQueryParams', () => {
```

Comment:
```suggestion
describe('normalizeQueryParams', () => {
```

Reply:
oops... sorry, thanks for noticing it :slightly_smiling_face: :pray: